### ISSUE TITLE
Please add >=python3.9 to requirements section

### ISSUE THREAD

**Issue by TDavLinguist** [plandes/zotsite#30]
Thank you for this wonderful tool! I would like to suggest that you add >=python3.9 as a requirement on the README.md page. I am still running Ubuntu 20.04 with Python3.8 as default and I kept getting tracebacks until I switched to a venv with python3.9. I think specifying the Python version would be helpful for people like me that are stuck in dependency Hell and are afraid of breaking their systems by installing a version of Python outside of the repos. 😅

**Comment by plandes** [comment]
Hi @TDavLinguist,

Sorry to hear you're having these dependency issues--they can be challenging.  The README page does have Python 3.10 and 3.11 (currently supported Python versions) as badges as the first elements after the package version badge.

Regarding worrying about breaking dependencies: I recommend creating a virtual environment specifically for the tool using:
```bash
<path to new version of python/bin/python -m venv <target installation>
<target installation>/bin/pip install zotsite
```

Both the Python version and installed packages are different in each virtual environment.  Then your old setup remains the same and invoke zotsite using the absolute path of the virtual install:
```bash
<target installation>/bin/zotsite [options]
```

I hope this is helpful.

**Comment by TDavLinguist** [comment]
@plandes 
My apologies for not being clear! I was able to install it successfully, even before I opened this issue. Your helpful instructions are exactly what I did. I also did not notice the Python 3.10 and 3.11 badges, so that was my fault. I will close the issue! Thanks so much, again. You have no idea how helpful this tool is for me, because I wanted a way to put all of my papers on my site easily. 😀
